stripe-core/src/test/java/com/stripe/android/core/frauddetection/FraudDetectionDataRequestTest.kt

```diff
@@ -33,19 +30,7 @@ class FraudDetectionDataRequestTest {
 
     @Test
     fun headers_shouldReturnExpectedMap() {
-        assertThat(request.headers)
```

Comment:
Reduce it to testing the stuff that’s actually unique to `FraudDetectionDataRequest`.